nemo_skills/dataset/emergent_tts/prepare.py

```
@@ -0,0 +1,238 @@
+from pathlib import Path
+
+
+SYSTEM_MESSAGE = "You are a helpful assistant."
+DEFAULT_DATASET = "bosonai/EmergentTTS-Eval"
+DEFAULT_SPLIT = "train"
+WV_MOS_URL = "https://zenodo.org/record/6201162/files/wav2vec2.ckpt?download=1"
+
+
+def _require_deps():
+    try:
+        import numpy as np  # noqa: F401
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)